packages/tradershub/src/components/Dialog/Dialog.tsx

```diff
@@ -56,7 +56,7 @@ const Dialog = ({ children, className, shouldPreventCloseOnEscape = false }: TDi
     return (
         <div
             className={qtMerge(
-                'flex flex-col gap-1200 mx-auto w-[328px] lg:w-[440px] p-1200 bg-background-primary-base rounded-400',
+                'flex flex-col gap-1200 mx-auto w-[auto] lg:w-[auto] p-1200 bg-system-light-primary-background rounded-400',
```

Comment:
change `w-[auto] ` to w-full

Reply:
The dialog should be apt to the content, not having its own width. Some dialogs are smaller